index.html

```diff
@@ -64,33 +63,60 @@ <h2>
     <p>
       TBW
     </p>
+    <p>
+      TBW: <dfn>Example credential types</dfn>
+    </p>
     <h2>
       Model
     </h2>
-    <dl class="definitions" data-sort="" data-cite="vc-data-model">
+
+    <p class="note" title="Definitions under discussion">
+The goal of the definitions in this section is to reuse or establish
+terminology that is common across a variety of digital credential formats and
+protocols. Discussions surrounding these definitions are active and
+the definitions are likely to change over the next several months.
+    </p>
+
+    <dl class="definitions" data-sort="" data-cite="vc-data-model-2.0">
       <dt>
         <dfn>Digital credential</dfn>
       </dt>
       <dd>
-        <p>
-          Is a [=verifiable credential=] about a person.
-        </p>
+        A cryptographically signed digital document containing one or more
+        [=claims=] made by an [=issuer=] about one or more [=subjects=].
```

Comment:
Could a digital credential not be a [=verifiable credential=] about...
* a company?
* a pedigreed pet? 
* a provenanced antique or artwork? 
* a vehicle or other titled or registered asset?

Reply:
To @samuelgoto point, we were purposely trying to restrict this to things about people. It might be good to go back to that and then broaden it in the future as we gain more implementation experience.

Reply:
> do you feel that the original definition ("a digital credential is a [=verifiable credential=] about a person") is incorrect? 

Yes, as @TallTed mentioned, it's incorrect because "digital credential" and "identity credential" are about more than just people. 

We could say "a **personal** credential is a [=verifiable credential=] about a person"... but then, like "identity credentials", if we start naming the WebIDL and parameters in the API around the concept of a "person" or "identity", we paint ourselves into a corner wrt. other digital credential types in the future (which we know there is interest in from the digital credentials ecosystem).

We could also say that "this specification is currently scoped to verifiable credentials related to people"?

I've made that change in 38139921ddc3b46b54051134b2674fbe4ab9a149. This allows us to focus on people in the spec, but allow for expansion into other things in the future. WDYT, @marcoscaceres, @samuelgoto, @TallTed ?

Reply:
"[This specification is focused on digital credentials pertaining to people.](https://github.com/WICG/digital-identities/commit/38139921ddc3b46b54051134b2674fbe4ab9a149)" is not quite "this specification is _currently_ scoped to verifiable credentials related to people". "Currently" is key to this statement.

I would be mostly OK with "VCDMv2 is scoped to digital credentials related to people; future versions are expected to broaden their scope to include digital credentials related to other entity types."

That said, please note that the [Traceability Vocabulary](https://w3c-ccg.github.io/traceability-vocab/) and [Traceability Interop](https://github.com/w3c-ccg/traceability-interop) Work Items of the [Credentials Community Group](https://www.w3.org/groups/cg/credentials/) is focused entirely on VCs related to non-people. Changing the scope of the VCDMv2 as described above would appear to make the Traceability Task Force's output invalid until VCDMv3, or only valid with VCs based on VCDMv1. Both of these are troublesome, to my thinking.

Reply:
D'oh! Sorry. Too many interleaved specs on my workbench!

Reply:
(That said, "currently" is what was [described here](https://github.com/WICG/digital-identities/pull/83#discussion_r1510007100), but not what was implemented in https://github.com/WICG/digital-identities/commit/38139921ddc3b46b54051134b2674fbe4ab9a149. I think "currently" or similar phrasing should be applied.)

Reply:
Also are we considering credentials that describe multiple subjects in the same credential here, as that appears what the definition implies? I know VC DM supports this, but it does bring about a whole different set of complexities and most other credential formats such as SD-JWT and mDocs have stayed away from this. I'd prefer we limit to one subject

```suggestion
        [=claims=] made by an [=issuer=] about one or more [=subject=].
```

Reply:
A driving license is about multiple subjects: the issuer, the driver, and the vehicle classes the driver is allowed to operate.